lib/babe/crypto.go

```diff
@@ -83,6 +84,36 @@ func checkPrimaryThreshold(randomness Randomness,
 	return inoutUint.Compare(threshold) < 0, nil
 }
 
+func claimSecondarySlot(randomness Randomness,
+	slot, epoch uint64,
+	authorities []types.Authority,
+	keypair *sr25519.Keypair,
+	authorityIndex uint32,
+) (*VrfOutputAndProof, error) {
```

Comment:
nit why not have just 2 long-ish-but-not-too-long lines of arguments (especially since `slot` and `epoch` are on the same line)?

Reply:
changed